libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx

```diff
@@ -12,11 +12,13 @@ export interface ButtonsActionsLogsProps {
 export function ButtonsActionsLogs(props: ButtonsActionsLogsProps) {
   const { refScrollSection, data, pauseLogs, setPauseLogs } = props
 
-  const downloadJSON = (event: MouseEvent) => {
+  const downloadJSON = () => {
```

Comment:
you can use `downloadjs` directly

Reply:
https://github.com/Qovery/console/pull/1316/commits/5f553326bc98c897a74a5871c5eefb98ecbea565